### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

**Review by mingazhev — APPROVED** [review]

File: HappyTravel.Edo.BookingStatusUpdate/Services/StatusUpdateService.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            _applicationLifetime = applicationLifetime;
+        }
+
+
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            var bookingList = await _edoClient.GetBookings();
+
+            foreach (var bookingId in bookingList)
+            {
```
Comment (kirillta) [inline HappyTravel.Edo.BookingStatusUpdate/Services/StatusUpdateService.cs:22]: Redundant braces

File: HappyTravel.Edo.BookingStatusUpdate/Infrastructure/EdoHttpClient.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+
+namespace HappyTravel.Edo.BookingStatusUpdate.Infrastructure
```
Comment (kirillta) [inline HappyTravel.Edo.BookingStatusUpdate/Infrastructure/EdoHttpClient.cs:7]: Why `BookingStatusUpdate`?

File: HappyTravel.Edo.BookingStatusUpdate/Services/StatusUpdateService.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.Edo.BookingStatusUpdate/Services/StatusUpdateService.cs]: Fixed

File: HappyTravel.Edo.BookingStatusUpdate/Infrastructure/EdoHttpClient.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.Edo.BookingStatusUpdate/Infrastructure/EdoHttpClient.cs]: Fixed

**Pull request merged**